api/src/main/java/org/pmiops/workbench/api/BigQueryService.java

```diff
@@ -85,6 +87,13 @@ public Date getDate(List<FieldValue> row, int index) {
         if (row.get(index).isNull()) {
             throw new BigQueryException(500, "FieldValue is null at position: " + index);
         }
-        return java.sql.Date.from(Instant.ofEpochMilli(Double.valueOf(row.get(index).getStringValue()).longValue() * 1000));
```

Comment:
For cohort materialization here's what I did:

https://github.com/all-of-us/workbench/blob/master/api/src/main/java/org/pmiops/workbench/cohortbuilder/FieldSetQueryBuilder.java#L494

That might be simpler?

Reply:
So before setting the timezone... intellij --> Sun Dec 09 18:00:00 CST 2001
command line --> Mon Dec 10 00:00:00 GMT 2001

Reply:
Wow. :)

The code I linked to above should return the same thing in either case, due to the withZoneUTC() thing here:

https://github.com/all-of-us/workbench/blob/master/api/src/main/java/org/pmiops/workbench/cohortbuilder/FieldSetQueryBuilder.java#L51

Reply:
Change method to return a string instead of a Date. Also, changed so that there is a separate method for Date and Timestamp.